hack/verify-govet.sh

```diff
@@ -1,83 +1,19 @@
 #!/bin/bash
 
+set -o errexit
 set -o nounset
 set -o pipefail
 
-GO_VERSION=($(go version))
-
-if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.[4-5]') && -z "${FORCE_VERIFY-}" ]]; then
-  echo "Unknown go version '${GO_VERSION}', skipping go vet."
-  exit 0
-fi
+echo $(go version)
 
 OS_ROOT=$(dirname "${BASH_SOURCE}")/..
-source "${OS_ROOT}/hack/common.sh"
-source "${OS_ROOT}/hack/util.sh"
 
 cd "${OS_ROOT}"
-mkdir -p _output/govet
-
-os::build::setup_env
-
-FAILURE=false
-test_dirs=$(find_files | cut -d '/' -f 1-2 | sort -u)
-for test_dir in $test_dirs
-do
-  go tool vet -shadow=false $test_dir
```

Comment:
It's not about the time it takes nor how many errors are reported. If we want to require no vet errors in PRs, then obviously we want `go vet` to return no output ;)

And removing shadow was explained in the commit message (my bad for not making it more visible):


----------------


According to Rob Pike:

  (...) the vet -shadow feature is disabled for a reason. It does
  not give the level of confidence the other analyses do.

(https://groups.google.com/d/msg/golang-nuts/fbxCYRPKTuM/6pKJ8qPdw6AJ)

We're getting a lot of false positives for legit code like:

	if err := ...; err != nil {
		...
	}

-------------------

And false positives can be seen in previous travis logs:

https://travis-ci.org/openshift/origin/builds/131434148

Notice that `go vet` in 1.5 doesn't report any error, while in 1.6 it reports many cases of the pattern above.